app/components/ClientSampleCode.tsx

```diff
@@ -403,5 +403,66 @@ export function ClientSampleCode({
           <Highlight language="sh" code="./a.out" />
         </>
       )
+    case 'c#':
+      return (
+        <>
+          In Visual Studio, create a new C# Console App under File {'>'} New{' '}
+          {'>'} Project. After the project initializes, right click the solution
+          in the Solution Explorer and click Manage NuGet packages for solution.
+          Browse for and install Confluent.Kafka. Copy the following into your
+          Progam.cs file.
+          <Highlight
+            language="cs"
+            code={dedent(String.raw`
+            using Confluent.Kafka;
+
+
+            var config = new ConsumerConfig
+            {
+              SecurityProtocol = SecurityProtocol.SaslSsl,
+              BootstrapServers = "kafka.${domain ?? 'gcn.nasa.gov'}",
+              GroupId = Guid.NewGuid().ToString(),
+              AutoOffsetReset = AutoOffsetReset.Earliest,
```

Comment:
Is this necessary, or is it the default?

Reply:
I'll remove that line in the next commit, it was a holdover from some sample code I was following, but not needed